tests/files/sym/test_multiplex_dump.sym

```diff
@@ -3,6 +3,7 @@ Title="Untitled"
 
 {SIGNALS}
 Sig=MultiplexedSig unsigned 8 /max:1
+Sig=NormalSig unsigned 8 /max:1
```

Comment:
I think that the changes to this file (and the ones in `test_database.py` where you check the `TestMultiplexer` message) are now redundant?

Reply:
Yes. I reverted the change on that sym file and updated the function `test_multiplex_sym_with_empty_signal_name_dump`.